javascript/javascript3/week1/giphy_searcher/js/app.js

```diff
@@ -0,0 +1,31 @@
+const text = document.getElementById('text')
```

Comment:
avoid name of var which are reserve keys in JS.

Reply:
Sur Naresh. I agree with you. I will change it.